cmd/auth.go

```diff
@@ -0,0 +1,119 @@
+package cmd
+
+import (
+	"errors"
+	"syscall"
+
+	"github.com/jsdelivr/globalping-cli/globalping"
+	"github.com/spf13/cobra"
+)
+
+func (r *Root) initAuth() {
+	authCmd := &cobra.Command{
+		Use:   "auth",
+		Short: "Authenticate with the Globalping API",
+		Long:  "Authenticate with the Globalping API for higher measurements limits.",
+	}
+
+	loginCmd := &cobra.Command{
+		RunE:  r.RunAuthLogin,
+		Use:   "login",
+		Short: "Log in to your Globalping account",
+		Long:  `Log in to your Globalping account for higher measurements limits.`,
+	}
```

Comment:
If the user is already logged in, silently call logout in the background to revoke the previous token.